features/other_a/reading.feature

```diff
@@ -11,7 +11,9 @@ Feature: Reading count
   Then I should see "Sorry, you don't have permission"
     And I should not see "History" within "div#dashboard"
   When I go to second_reader's reading page
-  Then I should see "History" within "div#dashboard"
+  Then I should see the page title "History | Example Archive"
```

Comment:
Just one more quick note! We prefer to leave the " | Example Archive" portion off so the tests don't fail for folks who have changed their dev environment's name.